schedule/views.py

```diff
@@ -393,7 +393,7 @@ def convert(ddatetime):
     # event, using the "event_id" or the occurrence with the specified "id".
     # for more info https://github.com/llazzaro/django-scheduler/pull/169
     i = 1
-    if Occurrence.objects.all().count() > 0:
+    if Occurrence.objects.all().exists():
```

Comment:
using `if queryset.exists()` is more efficient than checking `if queryset.count() > 0` [read more](https://django.doctor/advice/C9002)